org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/AbstractProcessDialog.java

```diff
@@ -630,9 +630,12 @@ protected void saveReportOption (){
 		if (!isReport() && !isJasperReport()){
 			return;
 		}
-		if(freportType.getSelectedItem() != null) {
+		if(freportType.getSelectedItem() != null && !Util.isEmpty(freportType.getSelectedItem().getValue())) {
 			getProcessInfo().setReportType(freportType.getSelectedItem().getValue().toString());
 		}
+		else {
+			getProcessInfo().setReportType(MSysConfig.getValue(MSysConfig.ZK_REPORT_TABLE_OUTPUT_TYPE, "HTML"));

```

Comment:
hmm ... can it be other report output type here, i.e ZK_REPORT_FORM_OUTPUT_TYPE or ZK_REPORT_FORM_OUTPUT_TYPE ?

Reply:
According to @norbertbede the these 2 SysConfigs define defaults per form, per table, but they should have the same attributes (HTML, PDF,...). The process audit intention is to catch, which type was taken as default or selected by user. 

In the latest commit I used the logic from ZkReportView.init() for setting the default Report Type